examples/todos/src/containers/AddTodo.spec.js

```diff
@@ -0,0 +1,34 @@
+import React from 'react';
+import { shallow } from 'enzyme';
+import toJson from 'enzyme-to-json';
+import AddTodo from './AddTodo';
+import configureStore from 'redux-mock-store';
+
+const setup = (setupProps = {}) => {
+  const store = configureStore()({ todos: [] });
+  const wrapper = shallow(<AddTodo store={store}/>);
+
+  return { store, wrapper };
+};
+
+describe('AddTodo component', () => {
+  it('render AddTodo component', () => {
+    const { wrapper } = setup();
+    expect(toJson(wrapper)).toMatchSnapshot();
```

Comment:
Maybe we don't need `toJson` here, I'm not sure, please try to remove it 😄 
`expect(wrapper).toMatchSnapshot();`